src/main/java/cz/muni/fi/dao/TeamDao.java

```diff
@@ -0,0 +1,48 @@
+package cz.muni.fi.dao;
+
+import cz.muni.fi.entities.Car;
+import java.util.List;
+
+/**
+ * @author Lucie Kureckova, 445264
+ */
+public interface TeamDao {
+    
+    /**
+     * Add team to DB
```

Comment:
Please add a blank line between description and other tags in javadoc. Oracle have it like this on their website - http://www.oracle.com/technetwork/articles/java/index-137868.html